content/influxdb/v2.0/tools/grafana.md

```diff
@@ -69,8 +69,64 @@ configure your InfluxDB connection:
 {{% tab-content %}}
 ## Configure Grafana to use InfluxQL
 
-With **InfluxQL** selected as the query language in your InfluxDB data source,
-configure your InfluxDB connection:
+1. [Set up the InfluxDB 1.x compatibility API](#set-up-the-influxdb-1x-compatibility-api)
+2. [Configure your InfluxDB connection](#configure-your-influxdb-connection)
+
+### Set up the InfluxDB 1.x compatibility API
+Grafana uses the [InfluxDB 1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/)
+to query InfluxDB 2.0 using InfluxQL.
+To successfully authenticate with and query the 1.x compatibility API:
+
+- [Create a InfluxDB v1 authorization](#create-an-influxdb-v1-authorization)
+- [Create a InfluxDB DBRP mapping](#create-an-influxdb-dbrp-mapping)
+
+#### Create an InfluxDB v1 authorization
+
+{{% note %}}
+Setting up an InfluxDB v1 authorization is only required if you need to authenticate
```

Comment:
a special v1 authentication mechanism exists in 2.0 to provide this compatibility for our 1.x users and tools.

Reply:
I think folks may be confused by the InfluxDB 2.0 username and password === which is used to log into InfluxDB 2.0, but has nothing to do with the actual v1 auth token created using a username/password combo

Reply:
Good point. I'll clarify.